### PR TITLE
fix(symfony): documentation request _format

### LINKED ISSUE #6372: Swagger Docs jsonopenapi format return HTML
**API Platform version(s) affected**: 3.3.3

**Description**  
Since upgrade from 3.2 to 3.3.3 the jsononpenapi docs url return HTML.

**How to reproduce**  
Just try https://127.0.0.1:8000/api/docs.jsonopenapi in local

### PR DESCRIPTION
**Pull request by soyuka**
fixes #6372

### CODE REVIEW

File: features/openapi/docs.feature [hunk 1]
```diff
@@ -432,3 +432,9 @@ Feature: Documentation support
     ]
     """
     And the JSON node "components.schemas.DummyBoolean.properties.isDummyBoolean.owl:maxCardinality" should not exist
+
+  Scenario: Retrieve the OpenAPI documentation in JSON
+    Given I add "Accept" header equal to "text/html,*/*;q=0.8"
```
Comment (soyuka) [inline features/openapi/docs.feature:437]: @bendavies note that browsers are sending `*/*` without that we always control the Accept header before the `.{_format}` and we'd get a 406.

Comment (bendavies) [reply features/openapi/docs.feature:437]: @soyuka i'm not sure what you mean sorry.

**Comment by bendavies** [comment]
@soyuka can we have a release for this? thanks!

**Comment by soyuka** [comment]
sorry tried tonight but wasn't able to (needed to catch up on api-platform/api-platfom) will try again tomorrow!

**Comment by soyuka** [comment]
done !

**Pull request merged**